liquid.ps1

```diff
@@ -1,29 +0,0 @@
-
```

Comment:
This file has a few references, they will need to be cleaned up:
```bash
± grep -RniI liquid.ps1
liquidprompt:4210:        # shellcheck source=liquid.ps1
docs/theme/default.rst:70:An example template file is available: liquid.ps1_.
docs/theme/default.rst:72:.. _liquid.ps1: https://github.com/nojhan/liquidprompt/blob/master/liquid.ps1
```

Reply:
I added themes/minimal/minimal.ps1, which should be less confusing. It is just default minus a couple of sections.